content/pricing/billing.textile

```diff
@@ -33,3 +33,17 @@ To update your billing details:
 You can update your payment details and view your invoices in this screen.
 
 It's also possible to have your invoices sent to an accounts department by filling in the *Optional billing email address*.
+
+h2(#alerts). Billing alerts
+
+Billing alerts notify you by email when your monthly spending exceeds a set amount on the accounts where you have paid "packages":/docs/pricing#packages (standard or pro). This feature helps you monitor costs and avoid unexpected charges.
+
+The following steps guide you through setting up billing alerts:
+
+* Log in to your Ably "account.":https://ably.com/login
+* Go to the *Account* menu in the top navigation bar.
+* Select *Billing*.
+* Scroll down to the *Billing alerts* section.
```

Comment:
We do have the route `https://ably.com/accounts/any/package#billing-alerts-section` available to simplify these steps

Reply:
Perfect for reducing these.

Reply:
I tested this URL: `https://ably-dev.com/accounts/any/package#billing-alerts-section`, and it worked... but only for `ably-dev.com`. Direct links to the dashboard are the approach we should aim for.

However, I noticed the dashboard copy doesn’t include ID tags, so I had to rely on the URL you shared, @matt423. How did you create this? I’ve updated it to just `ably-dev.com`. That said, I haven’t been able to test it yet, as I don’t have access to an active Standard or Pro account. So lastly, @matt423, can you help me check this link? Thanks :)

Reply:
The URL will work for ably.com too, so we can link to it confidently.  It follows the convention of other `any` URL's which uses the last account accessed